Tests/FluentAssertions.Specs/Collections/GenericCollectionAssertionOfStringSpecs.cs

```diff
@@ -1946,5 +1946,192 @@ public void When_string_collection_does_not_satisfy_all_inspectors_it_should_thr
         }
 
         #endregion
+
+        #region Ascending Order
+
+        [Fact]
+        public void When_strings_are_in_ascending_order_it_should_succeed()
+        {
+            // Arrange
+            string[] strings = { "alpha", "beta", "theta" };
+
+            // Act
+            Action act = () => strings.Should().BeInAscendingOrder();
+
+            // Assert
+            act.Should().NotThrow();
+        }
+
+        [Fact]
+        public void When_strings_are_not_in_ascending_order_it_should_throw()
+        {
+            // Arrange
+            string[] strings = { "theta", "alpha", "beta" };
+
+            // Act
+            Action act = () => strings.Should().BeInAscendingOrder("of {0}", "reasons");
+
+            // Assert
+            act.Should()
+                .Throw<XunitException>()
+                .WithMessage("Expected*ascending*of reasons*index 0*");
+        }
+
+        [Fact]
+        public void When_strings_are_in_ascending_order_according_to_a_custom_comparer_it_should_succeed()
+        {
+            // Arrange
+            string[] strings = { "alpha", "beta", "theta" };
+
+            // Act
+            Action act = () => strings.Should().BeInAscendingOrder(new ByLastCharacterComparer());
+
+            // Assert
+            act.Should().NotThrow();
+        }
+
+        [Fact]
+        public void When_strings_are_not_in_ascending_order_according_to_a_custom_comparer_it_should_throw()
+        {
+            // Arrange
+            string[] strings = { "dennis", "roy", "thomas" };
+
+            // Act
+            Action act = () => strings.Should().BeInAscendingOrder(new ByLastCharacterComparer(), "of {0}", "reasons");
+
+            // Assert
+            act.Should()
+                .Throw<XunitException>()
+                .WithMessage("Expected*ascending*of reasons*index 1*");
+        }
+
+        [Fact]
+        public void When_strings_are_in_ascending_order_according_to_a_custom_lambda_it_should_succeed()
+        {
+            // Arrange
+            string[] strings = { "alpha", "beta", "theta" };
+
+            // Act
+            Action act = () => strings.Should().BeInAscendingOrder((sut, exp) => sut.Reverse().First().CompareTo(exp.Reverse().First()));
```

Comment:
Out of curiosity, why do you write `Reverse().First()` instead of `Last()`?

Reply:
Because I forgot about `Last` 😜